micro-rdk-installer/Cargo.toml

```diff
@@ -14,23 +14,17 @@ version.workspace = true
 anyhow = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 crc32fast.workspace = true
-der.workspace = true
```

Comment:
Are any of the removals from this file the last ref to a package in the top-level `Cargo.toml`? If so they should probably be removed there too.

Reply:
yup, will do that